arch/arm/src/arm/arm_dataabort.c

```diff
@@ -70,15 +70,16 @@ void arm_dataabort(uint32_t *regs, uint32_t far, uint32_t fsr)
 {
   struct tcb_s *tcb = this_task();
 #ifdef CONFIG_LEGACY_PAGING
-  uint32_t *savestate;
+  bool savestate;
```

Comment:
Is arm_dataabort directly vectored into or called via arm_doirq?

Reply:
directly vectored